CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### Removed
 
 ### Bug Fixes
+- Changed watches.go to fix log.Info to modify message,and maxWorkers logic ([#2246](https://github.com/operator-framework/operator-sdk/pull/2246/files))
```

Comment:
I agree with @camilamacedo86, but I think it needs to be a bit more specific and user-friendly.

Also a few nits: the change description should end in a period with the link following in parentheses, and the link should be to the base PR URL, not to the `/files` subpath.

 How about this?
```suggestion
Fixed log formatting issue that occurred while loading the configuration for Ansible-based operators. ([#2246](https://github.com/operator-framework/operator-sdk/pull/2246))
```

Reply:
solved